source/Calamari.Tests/KubernetesFixtures/Scripts/KubernetesDeployment.ps1

```diff
@@ -263,6 +262,22 @@ function Deploy-CustomResources() {
 	return $DeploymentSuccess
 }
 
+function Write-CustomResourceErrorMessage() {
+	Param
+	(
+		[Parameter(ValueFromPipeline=$True, Position=0)]
+		[string] $json
+	)
+
+	Write-PlainError "`"kubectl apply -o json`" returned invalid JSON:"
+	Write-PlainError "---------------------------"
+	Write-PlainError "$json"
+	Write-PlainError "---------------------------"
+	Write-PlainError "This can happen with older versions of kubectl. Please update to a recent version of kubectl."
+	Write-PlainError "See https://github.com/kubernetes/kubernetes/issues/58834 for more details."
+	Write-PlainError "Custom resources will not be saved as output variables, and will not be automatically cleaned up."
+}
+
```

Comment:
As annoying as it is, I had to update this script so that it would produce the same output as the KubernetesDeployment.sh script. Unfortunately it was not producing the same output which is really annoying to test against.

Reply:
These updates could be ported back to server but I'm not sure its worth it as we'll be phasing out these script steps soon anyway. 🤷‍♂️